app/index.ts

```diff
@@ -0,0 +1,146 @@
+import express, { Express, Request, Response } from "express";
+import { fromHexAddress, getNetwork } from "./utils";
+import { Context, PrivkeyAccount, Client, RPCClient } from "firovm-sdk";
+import tokens from "./tokens.json";
+import "dotenv/config";
+
+const app: Express = express();
+const port = Number(process.env.PORT) || 8123;
+const bind = process.env.BIND || "0.0.0.0";
+const network = process.env.NETWORK || "Regtest"; // ["Mainnet", "Testnet", "Regtest"]
+const RPCURL = process.env.RPCURL || "http://guest:guest@127.0.0.1:8545";
+const PRIVKEY = process.env.PRIVKEY || "";
```

Comment:
Please move environments section to between line 5 and 7

Reply:
ok kub